src/content/en/tools/workbox/index.md

```diff
@@ -0,0 +1,177 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Service Worker Libraries.
+
+{# wf_published_on: 2015-01-01 #}
+{# wf_updated_on: 2017-05-16 #}
+
+# Workbox {: .page-title }
+
+Use [Workbox](https://workboxjs.org/) to simplify your development by making it
+easy to take advantage of powerful service worker features, eliminate boilerplate code, and automate service worker generation.
+
+<figure class="attempt-right">
+  <img src="/web/tools/workbox/thumb.png" alt="Workbo logo">
+</figure>
+
+Workbox is a collection of loosely-coupled libraries and tools, that focus on
+different service worker features and use-cases. The two core tools that Workbox provides are:
+
+**workbox-sw&mdash;**A service worker library to make managing fetch requests
+and caching as easy as possible.
+
+**workbox-cli&mdash;**A command-line tool to generate a service worker and a
+file manifest, that makes use of the workbox-sw module to become a fully-functioning service worker.
+
+These two high-level tools are built up from a number of lower-level modules
+that can be used independently or mixed-and-matched:
+
+**workbox-build&mdash;**A module that gan be used to generate a file manifest or
```

Comment:
For the rest of these you can leave out module and library.
* Generates a file manifest...
* Implements various ...